packages/core/package.json

```diff
@@ -3,32 +3,34 @@
   "version": "2.0.0-alpha.2",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build:lib": "rimraf lib && cross-env NODE_ENV=production babel -d lib/ src/",
-    "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config webpack.config.dist.js",
-    "cs-check": "prettier -l \"{playground,src,test}/**/*.js\"",
-    "cs-format": "prettier \"{playground,src,test}/**/*.js\" --write",
-    "dist": "npm run build:lib && npm run build:dist",
-    "lint": "eslint src test playground",
+    "build": "npm run build:cjs && npm run build:es && npm run build:es:lib",
+    "build:cjs": "NODE_ENV=production BABEL_ENV=cjs babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/cjs",
+    "build:es": "NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./dist/es",
+    "build:es:lib": "NODE_ENV=production BABEL_ENV=es babel --extensions \".js,.jsx,.ts,.tsx\" ./src --out-dir ./lib",
```

Comment:
@chanceaclark I had to add this line in because we needed to import things like `@rjsf/core/lib/utils`

Reply:
I will check that I still get a UMD Module, this is vital for me and perhaps other users... essentially I pull this from the build output

Reply:
@epicfaace I'm guessing that's to keep support for it. What's to stop people from using doing this now:
```tsx
import { deepEquals } from '@rjsf/core/dist/utils';`
```
IMO, it makes more sense to export the utils along with package or creating a utils package if you want to separate it.

Also, you are probably aware, but keep in mind this is still part of the alpha version so there is more freedom in introducing breaking changes.

Reply:
Yeah, good point. It seems like `react-jsonschema-form/lib/utils` is already used, though, see material-ui (https://github.com/rjsf-team/react-jsonschema-form/pull/1642/files#diff-92ef0ee17db18857941b12442a459de0R6) or #1643

Additionally, the documentation even has an example of using an actual component from the `lib` directory (https://github.com/rjsf-team/react-jsonschema-form/pull/1642/files#diff-443373237a67e0fe395c3da27e3595ebR553).

```
import SchemaField from "@rjsf/core/lib/components/fields/SchemaField";
```

This is probably bad, as we probably don't want to constrain ourselves with file structure. However, it would also be useful for users to be able to access the default fields / widgets. Perhaps it would be best to just make a `@rjsf/utils` package and expose fields through `@rjsf/core/fields` and widgets through `@rjsf/core/widgets`?

Reply:
I say we break out the utils package since it'll be better to maintain that way. If we need to make a path or minor to the utils package, in theory it shouldn't break anyones installs if they are using a `^` (i.e. `"@rjsf/utils": "^X.X.X"`).

For the fields and widgets, they should probably just be exposed in the root index file still so you can import them like so:
```tsx
import { ArrayField } from '@rjsf/core';`
```

We would just need to _ensure_ tree shaking is set up correctly, that way they don't import everything from `@rjsf/core`.

Reply:
Yeah, sounds good. Let's do that in another issue / PR.